slides.Rmd

```diff
@@ -463,6 +499,10 @@ borders %>%
 ```
 
 
+## ungroup()
+We need an example here of using ungroup using the example data - happy for suggestions.
+
+
 ## count()
```

Comment:
maybe this will be moved to the previous section in place of the base R functions?